fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccountTransactionRepository.java

```diff
@@ -44,4 +44,8 @@ List<SavingsAccountTransaction> findTransactionsAfterPivotDate(@Param("savingsAc
 
     @Query("select sat from SavingsAccountTransaction sat where sat.refNo = :refNo")
     List<SavingsAccountTransaction> findAllTransactionByRefNo(@Param("refNo") String refNo);
+
+    @Query("select sat from SavingsAccountTransaction sat where sat.savingsAccount.id = :savingsId and sat.dateOf <= :transactionDate and sat.reversed=0 ORDER BY sat.dateOf DESC")
```

Comment:
`sat.reversed=0` should be `sat.reversed=false`

Reply:
> `sat.reversed=0` should be `sat.reversed=false`

@galovics Comment Addressed, Please review.